nflow-rest-api-spring-web/src/main/java/io/nflow/rest/v1/springweb/ArchiveResource.java

```diff
@@ -0,0 +1,34 @@
+package io.nflow.rest.v1.springweb;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.nflow.engine.service.ArchiveService;
+import io.nflow.rest.v1.msg.ArchiveRequest;
+import io.nflow.rest.v1.msg.ArchiveResponse;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+
+@RestController
+@RequestMapping(value = "/nflow/v1/archive", produces = "application/json")
```

Comment:
use org.springframework.http.MediaType.APPLICATION_JSON_VALUE or APPLICATION_JSON_UTF8_VALUE constant for "application/json"